tests/endtoend/test_endtoend.py

```diff
@@ -56,26 +66,30 @@ def test_task_journey(driver, test_app):
     driver.find_element_by_id("add-item").click()
     driver.implicitly_wait(2)
     els = driver.find_elements_by_tag_name("td")
-    assert driver.find_element_by_id("todoname").text == "Movie on TV"
-    assert driver.find_element_by_id("tododesc").text == "To Do"
+    assert driver.find_element_by_id("todoname").text == "Watch movie"
```

Comment:
Minor, but the fact you had to change this in so many lines suggests it might be worth extracting into a `todo_title` variable or similar, and similarly for the description